account_analytic_distribution_widget_rebalance/static/tests/AnalyticDistribution.esm.js

```diff
@@ -0,0 +1,347 @@
+/** @odoo-module **/
+/*
+    Copyright 2025 Camptocamp SA (https://www.camptocamp.com).
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+*/
+
+import {click, getFixture} from "@web/../tests/helpers/utils";
+import {makeView, setupViewRegistries} from "@web/../tests/views/helpers";
+import {batchedOrmService} from "@analytic/services/batched_orm_service";
+import {registry} from "@web/core/registry";
+const {QUnit} = window;
+
+let serverData = {};
+let target = null;
+
+QUnit.module("Analytic", (hooks) => {
+    hooks.beforeEach(() => {
+        target = getFixture();
+        serverData = {
+            models: {
+                "account.analytic.account": {
```

Comment:
Hello, I'm not used to JS tests : why is this model declared with its entire name inside quotes, whereas the other ones have simplified names, without any quote ?

Reply:
It's just because it has `.` in its name, so we need to escape it 🤗

Reply:
Simply ! Thank you